nucypher/crypto/kits.py

```diff
@@ -76,3 +80,78 @@ def __init__(self, *args, **kwargs) -> None:
     def from_bytes(cls, some_bytes):
         capsule, sender_pubkey_sig, ciphertext = cls.split_bytes(some_bytes)
         return cls(capsule=capsule, sender_pubkey_sig=sender_pubkey_sig, ciphertext=ciphertext)
+
+
+Revocation = namedtuple('Revocation', ['prefix', 'arrangement_id', 'signature'])
```

Comment:
What do y'all think about this? I think it makes way more sense like this, but feels weird putting it outside of the `RevocationKit`.

I can either make those static methods class-methods, or leave this here, or something else entirely different.

Reply:
Yeah - this is feeling a bit awkward and informal out here...

Reply:
It can be moved inside the class, but I think I'd have to use a `classmethod` to use the methods I want.